docs/conf.py

```diff
@@ -111,6 +111,7 @@
         "binderhub_url": "https://mybinder.org",
         "colab_url": "https://colab.research.google.com/",
         "deepnote_url": "https://deepnote.com/",
+        "retrolite_url": "./lite/retro/notebooks/",
```

Comment:
Could we call this `jupyterlite_url` for now, since that is the "parent" tool, and document how to make it connect w/ retro vs. lab etc in the text below?

Reply:
I opened an issue some time ago in jupyterlite repo wrt to having a discrete *lite identities for these two UIs, motivated in part by this PR: https://github.com/jupyterlite/jupyterlite/issues/549

Reply:
I think where we're heading for the launch button config is to provide a list of configuration snippets that each map onto one button, so I'd imagine this will eventually look like:

```python
"launch_buttons": [
    {"type": "jupyterlite", "root": "/lite", "prefix": "/lab", "name": "Launch JupyterLite"},
    {"type": "jupyterlite", "root": "/lite", "prefix": "/retro", "name": "Launch RetroLite"},
    {"type": "jupyterhub", "url": "https://hub1.myorg.org", "name": "Launch JupyterHub One"},
    {"type": "jupyterhub", "url": "https://hub2.myorg.org", "name": "Launch JupyterHub Two"},
]
```

So that's why I was thinking we'd just have one name for it, does that make sense or do you think it would make more sense to use `type: "jupyterlite"` and `type: "retrolite"`?